docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -485,6 +499,18 @@ class Sidebar extends React.Component<any, any> {
           />
           <Text color="white" content={pkg.version} size="medium" styles={logoStyles} />
         </Segment>
+        <Menu vertical fluid pills styles={navBarStyles} items={withPrototypes} />
+        <Input
```

Comment:
did you consider using `<Dropdown>` component which already has the search functionality?

Reply:
+1 for `Dropdown`